SPECS/iputils/iputils.spec

```
@@ -58,32 +50,47 @@ mv -f RELNOTES.tmp RELNOTES.old
 %{_sbindir}/rdisc
 %{_sbindir}/ninfod
 %{_sbindir}/tracepath
-%{_sbindir}/traceroute6
+%{_sbindir}/tracepath6
 %{_bindir}/tracepath
-%{_bindir}/traceroute6
-%caps(cap_net_raw=p) %{_sbindir}/clockdiff
-%caps(cap_net_raw=p) %{_sbindir}/arping
+%caps(cap_net_raw=p) %{_bindir}/clockdiff
+%{_sbindir}/clockdiff
+%caps(cap_net_raw=p) %{_bindir}/arping
+%{_sbindir}/arping
 %caps(cap_net_raw=p cap_net_admin=p) %{_bindir}/ping
 %caps(cap_net_raw=p cap_net_admin=p) %{_bindir}/ping6
+%exclude %{_datadir}/locale/
+%exclude %{_sysconfdir}/init.d/ninfod.sh
 
 %changelog
-*   Wed Jul 29 2020 Andrew Phelps 20180629-5
--   Add ping6 symlink.
-*   Sat May 09 2020 Nick Samson <nisamson@microsoft.com> 20180629-4
--   Added %%license line automatically
-*   Mon Apr 13 2020 Jon Slobodzian <joslobo@microsoft.com> 20180629-3
--   Verified license. Removed sha1. Fixed commented out URL. Fixed formatting.
-*   Tue Sep 03 2019 Mateusz Malisz <mamalisz@microsoft.com> 20180629-2
--   Initial CBL-Mariner import from Photon (license: Apache2).
-*   Thu Sep 06 2018 Ankit Jain <ankitja@vmware.com> 20180629-1
--   Updated to version 20180629
-*   Wed Nov 16 2016 Alexey Makhalov <amakhalov@vmware.com> 20151218-4
--   Remove openssl and gnutls deps
-*   Tue May 24 2016 Priyesh Padmavilasom <ppadmavilasom@vmware.com> 20151218-3
--   GA - Bump release of all rpms
-*   Thu Apr 07 2016 Mahmoud Bassiouny <mbassiouny@vmware.com> 20151218-2
--   Fixing permissions for binaries
-*   Fri Jan 22 2016 Xiaolin Li <xiaolinl@vmware.com> 20151218-1
--   Updated to version 2.4.18
-*   Tue Oct 20 2015 Harish Udaiya Kumar <hudaiyakumar@vmware.com> 20121221-1
--   Initial build.    First version
+* Tue Feb 15 2022 Rachel Menge <rachelmenge@microsoft.com> - 20211215-1
+- Update source to 20211215
```

Maybe we should add the switch to `meson`
```suggestion
- Update source to 20211215
- Use `meson` instead of `make`
```